gems/pending/fs/fat32/directory_entry.rb

```diff
@@ -399,12 +404,12 @@ def mkLfn(name)
         LFN_NAME_COMPONENTS.each {|comp|
           chStart = (ent_num - 1) * CHARS_PER_LFN + comp[LFN_NC_OFFSET]
           if chStart > name.length
-            ent["#{comp[LFN_NC_HASHNAME]}"] = "\377" * (comp[LFN_NC_LENGTH] * 2)
+            ent["#{comp[LFN_NC_HASHNAME]}"] = "\377".b * (comp[LFN_NC_LENGTH] * 2)

```

Comment:
Yes, it was added in 2.0 or 2.1.

https://ruby-doc.org/core-2.2.0/String.html#method-i-b